src/macaron/policy_engine/policy.py

```diff
@@ -147,26 +146,227 @@ def parse(target: Any) -> bool:
             raise InvalidPolicyError(f"No support for policy {policy}")
 
 
+@dataclass
+class SoufflePolicy:
+    """
+
+    A high level for about the analysis described in souffle datalog.
+
+    Parameters
+    ----------
+    text: str
+        The full text content of the policy
+    sha: str
+        The sha256 sum digest for the policy text
+    _failed:
+        The list of repositories, policy pairs failing
+    _passed:
+        The list of repository, policy pairs passing
+
+    """
+
+    text: str
+    sha: str
+    filename: str
+    _result: dict | None
```

Comment:
I think putting the documentation here would be rendered in the Sphinx documentation for those private attributes. I think we should relocated it. How do you think?

Reply:
Fixed here https://github.com/oracle-samples/macaron/pull/46/commits/4e61112595339f7871796e707f604cc1ff1b5711